examples/ota-provider-app/ota-provider-common/OTAProviderExample.h

```diff
@@ -91,6 +91,17 @@ class OTAProviderExample : public chip::app::Clusters::OTAProviderDelegate
 
     void SetMaxBDXBlockSize(uint16_t blockSize) { mMaxBDXBlockSize = blockSize; }
 
+    uint32_t GetVendorId() { return mVendorId; }
+    uint32_t GetProductId() { return mProductId; }
+    short unsigned int GetHardwareVersion() { return mHardwareVersion; }
+    uint32_t GetSoftwareVersion() { return mRequestorSoftwareVersion; }
+    std::vector<chip::app::Clusters::OtaSoftwareUpdateProvider::DownloadProtocolEnum> GetProtocolsSupported()
```

Comment:
There's an esp32 impl of this, which means this is common code and std shouldn't be used. This should be ok as static - there aren't that many available protocols.